x-pack/plugins/security_solution/public/common/components/header_actions/add_note_icon_item.tsx

```diff
@@ -6,12 +6,34 @@
  */
 
 import React, { useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
 import { NotesButton } from '../../../timelines/components/timeline/properties/helpers';
 import { type TimelineType, TimelineTypeEnum } from '../../../../common/api/timeline';
 import { useUserPrivileges } from '../user_privileges';
-import * as i18n from './translations';
 import { ActionIconItem } from './action_icon_item';
 
+const NOTES_DISABLE_TOOLTIP = i18n.translate(
+  'xpack.securitySolution.timeline.body.notes.disableEventTooltip',
+  {
+    defaultMessage: 'Notes may not be added here while editing a template timeline',
```

Comment:
that's because this wasn't modified by this PR, this text existed long before the new notes system. Updated [here](https://github.com/elastic/kibana/pull/197312/commits/6630153159345bc043d1adbb25d9259818b47a13)